### PR TITLE
Adapt aquarius queries

### LINKED ISSUE #968: Adapt `Aquarius` queries for the new structure
Adapt and test the `Aquarius` queries so that the homepage/profile is working properly and we provide the teasers the proper data.

Done : 
- homepage doesn't crash and properly loads latest published list
- profile page loads properly `Published` tab
- search works

### LINKED ISSUE #969: Assets details page
Restore asset details page functionality only from a metadata perspective , ignore the actions tabs.

Done:
- `asset/{did}` loads properly and displays all the relevant metadata , what we have now.

### PR DESCRIPTION
**Pull request by claudiaHash**
- Fixes #968
- Fixes #969

Changes proposed in this PR:

- adapt queries to  aquarius v4 version and have 'Recently published' homepage section working
- render asset details page as much as possible without touching subgraph

### CODE REVIEW

File: apollo.config.js [hunk 1]
```diff
@@ -2,7 +2,7 @@ module.exports = {
   client: {
     service: {
       name: 'ocean',
-      url: 'https://subgraph.rinkeby.oceanprotocol.com/subgraphs/name/oceanprotocol/ocean-subgraph',
+      url: 'https://subgraphv4.rinkeby.oceanprotocol.com/subgraphs/name/oceanprotocol/ocean-subgraph',
```
Comment (kremalicious) [inline apollo.config.js:5]: oh boy, this is so tricky cause we ideally we now continue to have working test runs and builds but until we refactor all subgraph stuff for new subgraph, generating typings from the new subgraph should be postponed. So would suggest putting this back to the old Rinkeby subgraph, I just hope this doesn't affect the Aquarius part.

The goal is that e.g. locally for you `npm test` runs successfully.

Comment (claudiaHash) [reply apollo.config.js:5]: Sorry, I pushed it before I saw[ your comment](https://github.com/oceanprotocol/market/issues/968#issuecomment-1010979210). sure, I'll put back the old subgraph and only work on having 'Recently Published' section working for now.

**Comment by kremalicious** [comment]
OMG, I finally see all my test assets in this PRs preview, so great!

**Comment by kremalicious** [comment]
Aquarius became really good in telling us what's wrong with a query:

Screen Shot 2022-01-13 at 11 52 04

Current state of this PR is already super valuable for local development as we can finally reach the asset details page of our recently published assets. But pretty sure we only require some small tweak somewhere to get that search page working

**Review by kremalicious — APPROVED** [review]
tested a bunch of searches, and profile page, and asset details page and all seems working well

**Pull request merged**